### ISSUE TITLE
chroot: failed to run command 'mount': Exec format error

### ISSUE THREAD

**Issue by randomas** [portalmod/mod-iso#9]
Trying to build the iso running the create-live-iso.sh script I got a failure and the log file gave me:
chroot: failed to run command 'mount': Exec format error

TIA

**Comment by falkTX** [comment]
that can happen if you're running a 32bit system or 32bit distro, as it can't run 64bit chroots.
is this the case?

**Comment by randomas** [comment]
@falkTX Yes, thanks so much, that was indeed it, I changed it in the script to i386 and that got things going, now I have a problem with the kxstudio sources that seem to have been moved since the script was written, but I should be able to use the ones in the sources list from my system. I'll try and see if that works. 
Thanks again!
